confgenerator/testdata/invalid/windows/metrics-exporter_more_than_one_with_the_same_type/input.yaml

```diff
@@ -46,4 +46,4 @@ metrics:
     pipelines:
       default_pipeline:
         receivers: [hostmetrics,mssql,iis]
-        exporters: [google]
+        exporters: [google, google_1, google_2]
```

Comment:
Should we remove "google" from here? Seems like that one was a typo.

Reply:
Resolved offline. This is as expected.